src/Harness/Contracts/IParticipantRepository.cs

```diff
@@ -0,0 +1,9 @@
+using ScheduledJobs.Domain;
+
+namespace Harness.Contracts;
+
+public interface IParticipantRepository
+{
```

Comment:
Appreciate it's part of the harness, but should we not reuse contract from common for participant repo? (Sorry, not sure what this is actually doing)

Reply:
This is for adding mock users so there is no interface that will do this